crates/subspace-proof-of-time/src/pot.rs

```diff
@@ -0,0 +1,101 @@
+//! Common defines.
+
+use sp_core::{Blake2Hasher, Hasher, H256};
+use std::ops::BitXor;
+
+/// The 128 bit key for the AES encryption.
+#[derive(Clone, Debug)]
+pub struct AesKey(pub [u8; 16]);
+
+impl From<&AesSeed> for AesKey {
+    fn from(seed: &AesSeed) -> AesKey {
+        Self(h256_to_arr(Blake2Hasher::hash(&seed.0)))
+    }
+}
+
+/// Input to AES.
+#[derive(Clone, Debug)]
+pub struct AesSeed(pub [u8; 16]);
+
+impl From<H256> for AesSeed {
+    fn from(hash: H256) -> Self {
+        Self(h256_to_arr(hash))
+    }
+}
+
+/// Output from AES.
+#[derive(Clone, Debug)]
+pub struct AesCipherText(pub [u8; 16]);
+
+/// Config params for PoT.
+#[derive(Debug)]
+pub struct PotConfig {
```

Comment:
This looks like consensus logic, low-level library shouldn't really care about it.

Reply:
Clock maker, config moved to sc-subspace-consensus